### DISCUSSION TITLE
[outdated] Markor always modifies files, even in view mode

### DISCUSSION THREAD

**Discussion by Sjoerdfc** [gsantner/markor#1512] [Q&A] [answered]
Hi, I tried to search through release notes of previous releases, and in the issues and discussions, but I couldn't find anything about this problem I'm having. I'm using Markor 2.8.3 from F-Droid on Android 12.

Im actually not sure if its a bug or a feature. But it seems that since a few releases ago (I don't use Markor that often), Markor always modifies files. I have it set to "Prefer view", and I have my notes sorted by date so I can see which note I lasted edited. Recently, it seems Markor always modifies files, even though I only view them in view mode. So now everytime I view a file, markor modifies the file, and it's at the top of my notes (because sorted by date). So the sorted by date is kinda useless now.

I installed an older version, v2.5.0, and here it doesn't happen. It only modifies the file if you open it in edit mode, and actually change something.

Is this a bug? Or is it caused by a new feature, and can it maybe be turned off?

**Comment by gsantner** [comment, marked as answer]
@Sjoerdfc   
See this one: <https://github.com/gsantner/markor/issues/1494>

Generally, please use [Search](https://github.com/gsantner/markor/issues?q=)
